Formula/q.rb

```diff
@@ -1,30 +1,24 @@
-class Q < Formula
-  include Language::Python::Virtualenv
+# frozen_string_literal: true
 
+# Formula for q
```

Comment:
This shouldn't be needed

Reply:
thanks. removing (one of the brew testing commands failed because of this, but this was a while ago, so probably not needed anymore).